src/acq.c

```diff
@@ -75,7 +77,7 @@ bool acq_load(u32 count)
   nap_acq_load_wr_enable_blocking();
   nap_timing_strobe(count);
   if (chBSemWaitTimeout(&load_wait_sem, 1000) == RDY_TIMEOUT) {
-    printf("acq: Timeout waiting for sample load!\n");
+    log_warn("acq: Timeout waiting for sample load!\n");
```

Comment:
Do we have the capacity to put more info in the messages.  For instance, how long was the timout parameter here?